src/content/get-started/install/openshift.mdx

```diff
@@ -0,0 +1,204 @@
+
+By default, running privileged containers is not allowed in Red Hat OpenShift.
+This restriction affects the [Okteto Build](core/build-service.mdx) service.
+To grant privileged access to the Okteto Build service account, run:
+
+```bash
+ oc adm policy add-scc-to-user privileged -z okteto-buildkit -n okteto
+```
+
+This command grants the `privileged` SCC to the `okteto-buildkit` service account in the `okteto` namespace.
+
+### Grant access to the host to the Okteto Daemon service account
+
+By default, access to the host is restricted by default in Red Hat OpenShift.
+This restriction impacts the he [Okteto Daemon](self-hosted/helm-configuration.mdx#daemonset) service.
+To grant the necessary permissionsto the Okteto Daemon, run:
+
+```bash
+oc adm policy add-scc-to-user hostaccess -z okteto -n okteto
+```
+
+This command grants the `hostaccess` SCC to the `okteto` service account in the `okteto` namespace.
+
+### Copy Red Hat OpenShift wildcard certificate to the `okteto` namespace
+
+OpenShift provides the wildcard certificate for the wildcard domain `*.apps.<openshift-base-domain>`.
+We recommend reusing it when installing Okteto to [expose secure and unique endpoints](self-hosted/helm-configuration.mdx#subdomain) for your development environments.
+
+Copy the certificate to the `okteto` namespace by running:
+
+```yaml
+kubectl get secret <openshift-cluster-name>-primary-cert-bundle-secret -n openshift-ingress -o yaml | sed 's/namespace: openshift-ingress/namespace: okteto/' | kubectl apply -f -
+```
+:::info
+Note: The secret name `<openshift-cluster-name>-primary-cert-bundle-secret` is commonly used in OpenShift 4.x. Replace it with the actual secret name if it's different in your cluster.
+:::
+
+### Retrieve the Red Hat OpenShift Ingress Controller IP
+
+We recommend installing Okteto behind the Red Hat Openshift Ingress Controller.
+Obtain the IP address of your Red Hat OpenShift Ingress Controller by running:
+
+```bash
+kubectl get svc router-default -n openshift-ingress -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+```
+
+Store this value as `<ingress-controller-ip>`; you'll use it in the Okteto Helm configuration.
+
+
+## Installing Okteto
+
+Okteto is installed using a Helm chart. Follow these steps to install it.
+
+### Add the Okteto Helm repository
+
+Add the Okteto Helm repository and update it:
+
+```bash
+helm repo add okteto https://charts.okteto.com
+helm repo update
+```
+
+### Create the Helm configuration file
+
+Create a configuration file named `config.yaml` with the following content.
+Replace `<<okteto-license>>`, `<openshift-base-domain>`, `<openshift-cluster-name>`, and `<ingress-controller-ip>` with your actual values:
+
+```yaml
+license: <<okteto-license>>
+
+openshift:
+  enabled: true
+
+subdomain: apps.<openshift-base-domain>
+
+wildcardCertificate:
+  create: false
+  name: <openshift-cluster-name>-primary-cert-bundle-secret
```

Comment:
When I installed a cluster this was enough for me. 

But for the prospect, they had to add here the CA created automatically by openshift. They had to get the public key of the signing subordinate CA and add it following this steps: 

https://www.okteto.com/docs/1.18/self-hosted/install/certificates/#bring-your-own-certificate-authority

What I found in the Openshift docs to find the CA: 

_The Operator generates this signing certificate and puts it in a secret named router-ca in the openshift-ingress-operator namespace._

![image](https://github.com/user-attachments/assets/61d38e43-9bde-416f-8c0d-e04bb3183bac)

Reply:
It’s the same response as the comment on line 102